src/ContainerAppHelper.ts

```diff
@@ -0,0 +1,434 @@
+        try {
+            let command = `az containerapp update -n ${containerAppName} -g ${resourceGroup} -i ${imageToDeploy}`;
+            optionalCmdArgs.forEach(function (val: string) {
+                command += ` ${val}`;
+            });
+            await util.executeAndThrowIfError(command);
+        } catch (err) {
+            toolHelper.writeError(err.message);
+            throw err;
+        }
+    }
+
+    /**
+     * Updates an existing Azure Container App using the 'az containerapp up' command.
+     * @param containerAppName - the name of the existing Container App
+     * @param resourceGroup - the resource group that the existing Container App is found in
+     * @param imageToDeploy - the name of the runnable application image that the Container App will be based from
+     * @param optionalCmdArgs - a set of optional command line arguments
+     * @param ingress - the ingress that the Container App will be exposed on
+     * @param targetPort - the target port that the Container App will be exposed on
+     */
+    public async updateContainerAppWithUp(
+        containerAppName: string,
+        resourceGroup: string,
+        imageToDeploy: string,
+        optionalCmdArgs: string[],
+        ingress?: string,
+        targetPort?: string) {
+        toolHelper.writeDebug(`Attempting to update Container App with name "${containerAppName}" in resource group "${resourceGroup}" based from image "${imageToDeploy}"`);
+        try {
+            let command = `az containerapp up -n ${containerAppName} -g ${resourceGroup} -i ${imageToDeploy}`;
+            optionalCmdArgs.forEach(function (val: string) {
+                command += ` ${val}`;
+            });
+
+            if (!util.isNullOrEmpty(ingress)) {
+                command += ` --ingress ${ingress}`;
+            }
+
+            if (!util.isNullOrEmpty(targetPort)) {
+                command += ` --target-port ${targetPort}`;
+            }
+            await util.executeAndThrowIfError(command);
+        } catch (err) {
+            toolHelper.writeError(err.message);
+            throw err;
+        }
+    }
+
+    /**
+     * Updates an existing Azure Container App based from a YAML configuration file.
+     * @param containerAppName - the name of the existing Container App
+     * @param resourceGroup - the resource group that the existing Container App is found in
+     * @param yamlConfigPath - the path to the YAML configuration file that the Container App properties will be based from
+     */
+    public async updateContainerAppFromYaml(
+        containerAppName: string,
+        resourceGroup: string,
+        yamlConfigPath: string) {
+        toolHelper.writeDebug(`Attempting to update Container App with name "${containerAppName}" in resource group "${resourceGroup}" from provided YAML "${yamlConfigPath}"`);
+        try {
+            let command = `az containerapp update -n ${containerAppName} -g ${resourceGroup} --yaml ${yamlConfigPath}`;
```

Comment:
We need to add `--output none` to the end of this command call per the latest changes made in `v1`

Reply:
Updated.